### ISSUE TITLE
Kecerdasan buatan

### ISSUE THREAD

**Issue by slowy07** [bellshade/Python#246]
## Description
- implementasi artificial intelligence atau kercerdasan buatan pada bahasa pemograman python
- restruktur folder ``machine_learning`` yang kemudian dipindahkan ke ``implementation/artificial_intelligence``

contoh dari algoritma-algoritma yang digunakan pada kecerdasan buatan:
- [ ] LSTM (long short term memory network)
- [ ] normalization data transformation
- [ ] decision tree
- [ ] gaussian naive bayes
- [ ] gradient boosting
- [x] gradient descent
- [ ] k means clutsering
- [ ] k nearest neighbour
- [ ] linear discriminant analysis
- [ ] linear regression
- [ ] logistic regression
- [ ] multilayer preception classifier
- [ ] polymonial regression
- [ ] random forest classification
- [ ] fungsi scoring
- [ ] sequential minimum optimization
- [ ] similliar search
- [ ] SVM (support vector machine)
- [ ] frekuensi kata
- [ ] xgboost classifier
- [ ] xgboost regressor

## tambahan lainnya
- tidak diperkenankan pull request dengan file jupyter notebook file (``.ipynb``). 
- disarankan/diusahakan (opsional) meminimalisir penggunaan framework kecerdasan buatan seperti ``tensorflow`` ``pytorch``, atau ``sklearn``.
- diusahakan memberikan informasi atau dokumentasi kode atau komentar kode.

**Comment by kayabaakihiko13** [comment]
permisi kak,apakah boleh menambahkan lasso dan ridge regession untuk check list ini
soalnya itu marupakan linear regession yang bermain dengan OLS nya?
terima kasih atas perhatiannya

**Comment by slowy07** [comment]
boleh @kayabaakihiko13 👍🏻

**Comment by kayabaakihiko13** [comment]
permisi kak @slowy07  untuk normalisasi dan ridge apa nggak di centang kak?

**Comment by ShubhamZoro** [comment]
Want to add a random forest algorithm Please allow me to do so.

**Comment by slowy07** [comment]
go on @ShubhamZoro you can create issue and mention that pull request to you issue or this issue and we accept code with documentation with indonesia language, thanks!
